doc/conf.py

```diff
@@ -55,7 +55,8 @@ def __getattr__(cls, name):
                 'meliae', 'libnacl', 'decorator', 'libnacl.dual', 'libnacl.sign', 'libnacl.encode', 'libnacl.public',
                 'netifaces', 'ipv8.messaging.anonymization.tunnel', 'Tribler.community.gigachannel.community',
                 'networkx', 'validate', 'ipv8.REST.root_endpoint', 'anydex.restapi.root_endpoint',
-                'anydex.restapi.wallets_endpoint']
+                'anydex.restapi.wallets_endpoint', 'ipv8_service', 'faker', 'sentry_sdk',
```

Comment:
You can remove the `anydex.restapi.wallets_endpoint` mock (Anydex is not included in Tribler anymore) 👍

Reply:
Good point 👍 
Removed